aspnet/DatePicker/Server-Side-Events.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Validation
-description: Validate the date in server side
+title: Server Side Event in ASP.NET WebForm DatePicker control | Syncfusion
+description: Learn here all about Server side events in Syncfusion Essential ASP.NET WebForm DatePicker control, its elements, and more.
```

Comment:
Recheck on this.
![image](https://github.com/syncfusion-content/asp.netwebforms-docs/assets/95272441/6ba00726-e833-4955-82ae-034560fd5246)

Reply:
renamed